TreasureHuntGame/src/edu/sabanciuniv/asli/game/Game.java

```diff
@@ -160,6 +160,10 @@ public void play() {
 	
 	
 	private double calculateDistance() {
+
+		private sampleFunction() {
+			return xy;
+		}

```

Comment:
[Sev 2]: Nested methods are not allowed in Java. Consider moving 'sampleFunction' outside of 'calculateDistance'.
```
Category: code_quality - Severity: high
Line 164: private sampleFunction() {
```
**_Commented by gpt-4_**